access/src/main/java/edu/unc/lib/dl/ui/controller/RefreshMappingsController.java

```diff
@@ -47,10 +50,13 @@
     @Autowired
     private HeaderMenuSettings headerMenuSettings;
 
+    @Autowired
+    private GlobalPermissionEvaluator globalPermissionEvaluator;
+
     @RequestMapping(method = RequestMethod.GET)
     public void handleRequest(HttpServletRequest request, HttpServletResponse response) throws Exception {
-        AccessGroupSet accessGroups = getUserAccessGroups(request);
-        if (!accessGroups.contains(AccessGroupConstants.ADMIN_GROUP)) {
+        AccessGroupSet principals = getAgentPrincipals().getPrincipals();
+        if (globalPermissionEvaluator.hasGlobalPermission(principals, Permission.reindex)) {
```

Comment:
i'm confused. why are we throwing an exception when there IS a global permission?